src/function_send.c

```diff
@@ -47,6 +47,7 @@
 //#define 128 
 
 extern int number;
+extern unsigned char packet[10000];
```

Comment:
This will again cause build fail with LTO.

Reply:
corrected.